var/spack/repos/builtin/packages/gnina/package.py

```diff
@@ -65,5 +65,7 @@ class Gnina(CMakePackage, CudaPackage):
     def cmake_args(self):
         args = [
             "-DBLAS=Open",  # Use OpenBLAS instead of Atlas' BLAS
+            f"-DRDKIT_INCLUDE_DIR={self.spec['rdkit'].prefix.include}/rdkit",
```

Comment:
You could avoid the explicit `os.sep` using:

```suggestion
            "-DRDKIT_INCLUDE_DIR=" + self.spec['rdkit'].prefix.include.rdkit,
```